src/Microsoft.Identity.Web/AppServicesAuth/AppServicesAuthenticationTokenAcquisition.cs

```diff
@@ -49,16 +52,17 @@ private async Task<IConfidentialClientApplication> GetOrCreateApplication()
             {
                 ConfidentialClientApplicationOptions options = new ConfidentialClientApplicationOptions()
                 {
-                    ClientId = AppServiceAuthenticationInformation.ClientId,
-                    ClientSecret = AppServiceAuthenticationInformation.ClientSecret,
-                    Instance = AppServiceAuthenticationInformation.Issuer,
+                    ClientId = AppServicesAuthenticationInformation.ClientId,
```

Comment:
we should consider some refactoring in tokenAcqusition so we don't have to repeat this code. not something for this PR, but something for the future we shoudl consider.